apps/graphql/src/apps/itinerary/Itinerary.js

```diff
@@ -111,3 +127,13 @@ export type ApiResponseType = {|
     +booking_token: string,
   |}>,
 |};
+
+export type ValidityApiResponseType = {|
+  +flights_checked: boolean,
```

Comment:
I think all data from API should be optional:
```
+flights_checked?: boolean,
```

Reply:
True, updated.